dedupe/record_linkage_testing/README.md

```diff
@@ -0,0 +1,42 @@
+# Record Linkage Performance Testing
+
+This repository contains a performance testing project aimed at evaluating the scalability and responsiveness of the [record-linkage](https://github.com/CDCgov/phdi/tree/main/containers/record-linkage) API using [OpenTelemetry](https://opentelemetry.io/) for instrumentation. By utilizing Docker Compose, it provides an easy-to-use environment setup for running performance tests against the target API.
+
+## Prerequisites
+
+Before getting started, ensure you have the following installed:
+
+- Docker
+- Docker Compose
```

Comment:
Might be worth including a link to documentation on how to install for the different operating systems. [This link](https://docs.docker.com/engine/install/) might be sufficient.